code/style.css

```diff
@@ -1,5 +1,360 @@
-/* If you can see a peach background, the css file is correctly linked. Feel free to change or remove this styling! */
+html,
+body {
+  height: 100%;
```

Comment:
You rarely need to put height-measurements to big sections on your website. Unless your content is smaller than the screen, the body will stretch to fit everything in it. 😊